.github/workflows/release.yml

```diff
@@ -88,14 +81,19 @@ jobs:
           restore-keys: |
             download-unpacked-cache-
 
+      - name: Perform docker pull
+        if: ${{ github.event.inputs.performRelease == 'false' }}
+        run: |
+          docker pull quay.io/eclipse/che-plugin-registry:${{ github.event.inputs.version}} 
       - name: Perform release
+        if: ${{ github.event.inputs.performRelease == 'true' }}
         run: |
-          git config --global user.name "Mykhailo Kuznietsov"
-          git config --global user.email "mkuznets@redhat.com"
+          # git config --global user.name "Mykhailo Kuznietsov"
+          # git config --global user.email "mkuznets@redhat.com"
 
-          export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
+          # export GITHUB_TOKEN=${{ secrets.CHE_BOT_GITHUB_TOKEN }}
 
-          ./make-release.sh --version ${{ github.event.inputs.version}} --trigger-release
+          # ./make-release.sh --version ${{ github.event.inputs.version}} --trigger-release
```

Comment:
you've disabled the release here even when `performRelease == 'true'`. Did you mean to comment out all these steps?